### ISSUE TITLE
Dependency structure

### ISSUE THREAD

**Issue by krlmlr** [ropenscilabs/tic#26]
Any package that is imported by tic can't be updated easily on Windows. This means we need to be very careful with our dependencies.

- provide a wrapper around `remotes::install_deps()`? 
- run each step in its own process?
- parse `tic.R` only once, so that parsed representation doesn't require R6?
- ...?

**Comment by krlmlr** [comment]
Closing in favor of #35.
